### PR TITLE
在 latex 里面生成官方的本科生论文的封面（中文）。并且将页眉的徽标换成了红色的。

### PR DESCRIPTION
**Pull request by shaheming**
cover
banner

### CODE REVIEW

File: thesis.tex [hunk 1]
```diff
@@ -67,10 +67,10 @@
 
 %% 正文内容
 \pagestyle{main}
-\include{tex/intro}
-\include{tex/example}
-\include{tex/faq}
-\include{tex/summary}
+% \include{tex/intro}
```
Comment (gaocegege) [inline thesis.tex:70]: There is no need to comment the code here, IMO.

File: thesis.tex [hunk 2]
```diff
@@ -79,10 +79,10 @@
 \renewcommand\thealgorithm{\Alph{chapter}--\arabic{algorithm}}
 
 %% 附录内容，本科学位论文可以用翻译的文献替代。
-\include{tex/app_setup}
-\include{tex/app_eq}
-\include{tex/app_cjk}
-\include{tex/app_log}
+% \include{tex/app_setup}
```
Comment (gaocegege) [inline thesis.tex:82]: There is no need to comment the code here, IMO, too.

**Review by gaocegege — COMMENTED** [review]
@xplorld

WDYT, It has same feature with your PR: #199

**Comment by gaocegege** [comment]
Thanks for your contributions! It is awesome. 

Although I have some comments about the details, feel free to express your opinions

**Comment by xplorld** [comment]
@gaocegege not sure whether we should use a latex-generated one or a pdf. Definitely latex one (this pr) seems to be more beautiful.

File: sjtuthesis.cfg [hunk 3]
```diff
@@ -16,14 +16,15 @@
 %%
 %% labels in the title page
 %%
-\def\sjtu@label@major{专业}
-\def\sjtu@label@title{论文题目}
-\def\sjtu@label@author{论文作者}
+\def\sjtu@label@major{专业：}
+\def\sjtu@label@title{论文题目：}
+\def\sjtu@label@author{学生姓名：}
```
Comment (gaocegege) [inline sjtuthesis.cfg:21]: 我们需要变更 config 里的中文么

**Comment by gaocegege** [comment]
The PR is stale and we have already support title page in #242 , then I will close the PR soon. But, thanks for your contribution :-)

:tada:

**Comment by gaocegege** [comment]
如果你有新的问题或者觉得现在的实现还有问题可以提新的 PR :smiley: 谢谢你的贡献

**Pull request closed without merging**